admin/services/queue-management.service.js

```diff
@@ -31,6 +31,13 @@ const service = {
     const activeMessageCount = await serviceBusQueueAdminService.getQueueActiveMessageCount(queueName)
     if (activeMessageCount === 0) return
     return serviceBusQueueAdminService.clearQueue(queueName, activeMessageCount)
+  },
+
+  sendServiceBusQueueMessage: async function sendServiceBusQueueMessage (queueName, message, contentType) {
+    if (contentType === 'application/json') {
+      message = JSON.parse(message)
```

Comment:
Parsing could fail if the user supplied bad JSON.  Could you catch the error and bail out?